src/podman.scss

```diff
@@ -8,11 +8,22 @@
     @extend .ct-card;
 }
 
-// Decrease padding for the image/container toggle button list
 #containers-images, #containers-containers {
+    // Decrease padding for the image/container toggle button list
     .pf-c-table.pf-m-compact .pf-c-table__toggle {
         padding-left: 0px;
     }
+
+    // Ensure there's adaquate space between detail groups
+    .ct-listing-panel-body > .pf-l-flex {
+        gap: var(--pf-l-flex--spacer--lg);
```

Comment:
can't we give the spacer size as an option to a `Flex` component?

Reply:
Nope.

PatternFly does not support a gap option natively, as their browser support is wider than ours and they still want to support an older Safari version (which was the last holdout that didn't support gap where every other browser did for quite some time). Hence, using it here.

PatternFly uses margins instead, which are always present and not contextual like gaps are, so you'd end up with extra space to the side(s) and below.

Reply:
Thanks for the explanation!